components/RuiBreadcrumbs/src/RuiBreadcrumbs.ts

```diff
@@ -0,0 +1,67 @@
+/**
+* Copyright Deloitte Digital 2019
+*
+* This source code is licensed under the BSD-3-Clause license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import { LitElement, html, property, CSSResultArray, TemplateResult} from 'lit-element';
+import { variables, layout } from './RuiBreadcrumbs.css'
+
+
+export class RuiBreadcrumbs extends LitElement {
+	/**
+	 * The array of breadcrumbs
+	 */
+
+	@property({type : String})
+	public crumbs?: string = '';
+
+
+	/**
+    *
+    * The styles for breadcrumbs
+    * @remarks
+    * If you are extending this class you can extend the base styles with super. Eg `return [super(), myCustomStyles]`
+    */
+    public static get styles(): CSSResultArray {
+        return [variables, layout];
+    }
+
+    /* #endregion */
+
+
+    /* #region Methods */
+
+    /**
+    * Render method
+        * @slot This is a slot test
+    */
+
+    public render(): TemplateResult {
+		if (this.crumbs) {
+			const crumbsArray = JSON.parse(this.crumbs);
+			const activeCrumb = crumbsArray.pop();
+
+			return html`
+				<nav aria-label="Breadcrumb" class="crumbs">
```

Comment:
I'm not sure where they're coming from but there are few unnecessary comments getting rendered.
Can you please remove them?
<img width="348" alt="image" src="https://user-images.githubusercontent.com/816657/59811892-373db080-934f-11e9-9797-ef4db896598e.png">

Reply:
@ttaiyo I noticed these too with my work! Maybe it's a Lit Element thing, @grmillsy1 I'll be interested to see what the purpose of them are/if we can disable them

Reply:
@ttaiyo @jnelssonsmith looks like Lit-html uses comments as markers in their template.ts file to track the location of 'dynamic parts' to so it knows when/where it needs to update. 

I don't think there is a way to turn them off but can look more into it.